### DISCUSSION TITLE
What is the difference between Airbyte and Airbyte-Platform repository?

### DISCUSSION THREAD

**Discussion by laglangyue** [airbytehq/airbyte#33482] [Connector Questions]
What is the difference between Airbyte and Airbyte-Platform repository?

**Comment by pgrant87** [comment]
Hi @laglangyue 

We split these into 2 repos earlier this year as per our release note: https://docs.airbyte.com/release_notes/february_2023

> Improved the Airbyte GitHub repository structure and processes by splitting the current repo into two repos, `airbytehq/airbyte` for connectors and `airbytehq/airbyte-platform` for platform code.
Allows for isolated changes and improvements to the development workflow.
Simplifies the deployment process both internally and externally.
